schemas/oval/5.11.3/independent-definitions-schema.xsd

```diff
@@ -2158,13 +2158,24 @@
                                     </xsd:element>
                                     <xsd:element name="yamlpath" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="1">
                                           <xsd:annotation>
-                                                <xsd:documentation>Specifies an YAML Path expression to evaluate against the YAML file specified by the filename entity. This YAML Path expression must evaluate to a list of zero or more text values which will be accessible in OVAL via instances of the value_of entity. Any results from evaluating the YAML Path expression other than a list of text strings (e.g., a nodes set) is considered an error. The intention is that the text values be drawn from instances of a single, uniquely named element or attribute. However, an OVAL interpreter is not required to verify this, so the author should define the YAML Path expression carefully. Note that "equals" is the only valid operator for the yamlpath entity.</xsd:documentation>
+                                                <xsd:documentation>Specifies an YAML Path expression to evaluate against the YAML file specified by the filename entity. This YAML Path expression must evaluate to a list or a map (part of a map) of zero or more scalar values which will be accessible in OVAL via instances of the result entity. Any results from evaluating the YAML Path expression other than a list of scalar values (or a map with scalar values) at the bottom level (e.g. list of lists, list of maps, map with maps or lists) is considered an error. The intention is to drawn scalar values from instance(s) of a single (or multiple) flat element(s). However, an OVAL interpreter is not required to verify this, so the author should define the YAML Path expression carefully. Note that "equals" is the only valid operator for the yamlpath entity.</xsd:documentation>
                                           </xsd:annotation>
                                     </xsd:element>
-                                    <xsd:element name="value_of" type="oval-def:EntityStateAnySimpleType" minOccurs="0" maxOccurs="1">
+                                    <xsd:element name="result" type="oval-def:EntityStateRecordType" minOccurs="0">
```

Comment:
The OVAL schema specifies that name of the `field` should be lowercase and unique within a record.
https://github.com/OVALProject/Language/blob/master/docs/oval-definitions-schema.md#-entityobjectfieldtype-

Could we have yaml files with upper case letters in the keys?
I guess the `name` attribute in the `field` could have it  lowercase'd.
But what if the mapping already has the lowercased key? (Is it a too "corney" case?)

Reply:
I removed that requirement in this PR. It'd also be proposed to the OVAL community to remove this restriction (the alternative is a new CaseSensitiveField type or something like that).